docs/guides/latest/advance-use-case/instant-backup.md

```diff
@@ -0,0 +1,378 @@
+    - --enable-status-subresource=true
+    - --use-kubeapiserver-fqdn-for-aks=true
+    - --enable-analytics=true
+    - --logtostderr=true
+    - --alsologtostderr=false
+    - --v=3
+    - --stderrthreshold=0
+    env:
+    - name: NODE_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: spec.nodeName
+    - name: POD_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.name
+    image: suaas21/stash:restore-pvc-tpl_linux_amd64
+    imagePullPolicy: IfNotPresent
+    name: stash
+    volumeMounts:
+    - mountPath: /etc/stash
+      name: stash-podinfo
+    - mountPath: /etc/stash/repository/secret
+      name: stash-secret-volume
+    - mountPath: /tmp
+      name: tmp-dir
+    - mountPath: /source/data
+      name: source-data
+    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+      name: default-token-dp4pv
+      readOnly: true
+  volumes:
+  - name: source-data
+    persistentVolumeClaim:
+      claimName: source-data
+  - emptyDir: {}
+    name: tmp-dir
+  - downwardAPI:
+      defaultMode: 420
+      items:
+      - fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.labels
+        path: labels
+    name: stash-podinfo
+  - name: stash-secret-volume
+    secret:
+      defaultMode: 420
+      secretName: gcs-secret
+  - name: default-token-dp4pv
+    secret:
+      defaultMode: 420
+      secretName: default-token-dp4pv
+  ...
+...
+```
+
+**Verify CronJob:**
+
+It will also create a `CronJob` with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
+
+Verify that the `CronJob` has been created using the following command,
+
+```console
+$ kubectl get backupconfiguration -n demo 
+NAME                TASK   SCHEDULE       PAUSED   AGE
+deployment-backup          */40 * * * *            6m41s
+```
+
+## Backup Instant
+
+Now, we will take instant backup of the data volumes. for that, we will deploy a `BackupSession` object to trigger instant backup.
+
+**Create BackupSession:**
+
+Below is the YAML of the `BackupSession` crd that we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupSession
+metadata:
+  labels:
+    stash.appscode.com/backup-configuration: deployment-backup
+  name: deployment-backupsession
+  namespace: demo
+spec:
+  backupConfiguration:
+    name: deployment-backup
+```
+
+- `metadata.labels` holds the respective `BackupConfiguration` name as label. Stash backup sidecar container use this label to watch only the BackupSessions of that `BackupConfiguration`. The form of label must follow the following convention, `stash.appscode.com/backup-configuration: <BackupConfiguration name>`.
+
+Let's create the `BackupSession` object that we have have shown above,
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/advance-use-case/instant-backup/backupsession.yaml
+backupsession.stash.appscode.com/deployment-backupsession created
+```
+
```

Comment:
If everything goes well, the `stash` sidecar inside the Deployment will take a backup instantly.